docs/interfaces/pyapi/pyapi.rst

```diff
@@ -0,0 +1,425 @@
+.. highlight:: none
+.. _sec-interfaces-pyapi:
+
+#########################
+Python Interface - ctypes
+#########################
+
+ctypes is a foreign function library for Python. It provides C compatible data
+types, and allows calling functions in DLLs or shared libraries. It can be used
+to wrap these libraries in pure Python (cf. `ctypes documentation 
+<https://docs.python.org/3/library/ctypes.html>`_).
+
+Before going through the following sections, please make sure that you have
+installed a working version of the ctypes package, using your preferred
+installation procedure. Also, consider adding the Python file
+`tools/pythonapi/dftbplus.py`, which contains the actual ctypes interface,
+to your PYTHONPATH environment variable or symlink it. The scripts below assume
```

Comment:
Adding directories into the PYTHONPATH is considered bad practise, there should be a way to install it via `setup.py` or locally with `pip install -e .` instead.

Reply:
That's right. Given that the _dptools_ also use the setup provided by _distutils.core_, I think it would make sense to also go this way. I will try to set it up.